nntrainer/models/neuralnet.cpp

```diff
@@ -632,10 +632,11 @@ int NeuralNetwork::train_run() {
       layer_->net_hidden[0]->getGradientRef() = label;
     }
 
-    if (layer_->getType() == "RNN"){
-      Tensor &var_ref= layer_->net_hidden[0]->getVariableRef();
+    if (layer_->getType() == "RNN") {
+      Tensor &var_ref = layer_->net_hidden[0]->getVariableRef();
```

Comment:
I wonder if we can handle this on `RNN::initialize()`.

Reply:
removed